checkTranslations.mjs

```diff
@@ -0,0 +1,171 @@
+import parser from '@babel/parser';
+import traverse from '@babel/traverse';
+import mongodb from 'mongodb';
+
+import { resolve } from 'path';
+import { promises } from 'fs';
+
+async function getFiles(dir) {
+  const dirents = await promises.readdir(dir, { withFileTypes: true });
+  const files = await Promise.all(
+    dirents.map(dirent => {
+      const res = resolve(dir, dirent.name);
+      return dirent.isDirectory() ? getFiles(res) : res;
+    })
+  );
+  return Array.prototype
+    .concat(...files)
+    .filter(
+      file =>
+        !file.match('.spec') &&
+        (file.endsWith('.js') || file.endsWith('.ts') || file.endsWith('.tsx'))
```

Comment:
While the project does not seem to contain any, I would still add jsx as well, just to be safe.

Reply:
done